bubble_sort.s

```diff
@@ -0,0 +1,138 @@
+# This is realisation of basic bubble sort algorithm.
+# Prototype: void bubble_sort(int* arr , int num)
+# num - number of elements in array, num < 536870911
+# arr - pointer to the array
+# Elements of array could be signed integers
+# Both arguments must be sent in $a0 (arr) and $a1 (num)
+
+
+
+.globl main
+.data
+ 	numberofelem: .word numberofelem_data
+	insertarr: .word insertarr_data
+	resbody: .word resbody_data
+	res: .word res_data
+ 
+	numberofelem_data: .asciiz "Number of elements in array: "
+	insertarr_data: .asciiz "Elements of array:"
+	resbody_data: .asciiz "Sorted array: "
+	res_data: .asciiz " "
+
+	num: .word 0
+.text
+main:
+	# printf numberofelem
+	la      $t0, numberofelem 	# load address of msgprompt into $t0
+  	lw      $a0, 0($t0)       	# load data from address in $t0 into $a0
+  	li      $v0, 4            	# code of print_string
+  	syscall                   	# printf syscall
+	
+	# scanf number of elements
+	li 	$v0 , 5 		# code of read_int
+	syscall				# call scanf for int
+	sw 	$v0 , num		# store to num
+
+	# alloc memory for num integers
```

Comment:
Also, we do not have an `sbrk` call.
While `scanf` is expected by the end of 2018, `sbrk` is a little bit harder to crack.

I suggest to comment the scanf/sbrk code and to use a static array with some predefined data.